app/api/activitylog/activitylogMiddleware.js

```diff
@@ -27,8 +27,8 @@ export const BODY_REQUIRED_ENDPOINTS = [
 function mustBeLogged(baseurl, method, body) {
   const isLoggedRequest =
     baseurl.includes('/api/') &&
-    !ignoredMethods.includes(method) &&
-    !IGNORED_ENDPOINTS.includes(baseurl);
+    ((!ignoredMethods.includes(method) && !IGNORED_ENDPOINTS.includes(baseurl)) ||
```

Comment:
I think this solution is too specific for the problem, if we found more scenarios like this we dont want to have a long list of "ifs" here.

Probaly making the ignored endpoints a combination of method + endpoint is better.